src/components/Mapbox.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import ReactMapboxGl, {
+  ZoomControl,
+  Source,
+  Feature,
+  Layer
+} from 'react-mapbox-gl';
+
+import { circlesSource, uniqueCoordinates, lineSource } from '../utils/data';
+import { FilterValue } from './Filter';
+
+const Map = ReactMapboxGl({
+  accessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN!
+});
+
+interface Props {
+  data: any;
+  filters: Array<FilterValue>;
+}
+
+function Mapbox({ data, filters }: Props) {
+  const sensorCoordinates = uniqueCoordinates(data);
+  const [blastLinesFilter, setBlastLinesFilter] = useState(['all'] as any[]);
+
+  useEffect(() => {
+    const filtersToApply: any[] = ['all'];
+    filters.forEach(filter => {
+      const found = filtersToApply.find(f => f[1] === filter[0]);
+      if (found) {
+        found.push(filter[1]);
+      } else {
+        filtersToApply.push(['in', ...filter]);
+      }
+    });
+    setBlastLinesFilter(filtersToApply);
+  }, [filters]);
+
+  return (
+    <Map
```

Comment:
@karimkawambwa Just a heads up ....getting this `Style prop value must be an object  react/style-prop-object` on the console. Fixing this before approval

Reply:
@karimkawambwa Done, otherwise everything looks good